go-controller/pkg/clustermanager/egressip_controller.go

```diff
@@ -396,6 +365,8 @@ func newEgressIPController(ovnClient *util.OVNClusterManagerClientset, wf *facto
 		EIPClient:          ovnClient.EgressIPClient,
 		CloudNetworkClient: ovnClient.CloudNetworkClient,
 	}
+	markAllocator, _ := getEgressIPMarkAllocator()
```

Comment:
why do you discard the error ? shouldn't you handle it ?

Reply:
ya, true... i should refactor the func instead of covering it up..

Reply:
Done